csrc/executor.cpp

```diff
@@ -171,7 +171,7 @@ std::string FusionExecutor::getStructuredCode(
   if (isDebugDumpEnabled(DebugDumpOption::CudaToFile) ||
       isDebugDumpEnabled(DebugDumpOption::DebugInfo)) {
     std::stringstream file_name;
-    file_name << "__tmp_kernel" << getGlobalFusionCount() << ".cu";
+    file_name << "__tmp_kernel_" << kernel_id_ << ".cu";
```

Comment:
Would it be easier to keep the old `kernel934` style without the trailing underscore? I think the current special handling on diff-bench relies on this naming without trailing underscore, but we can definitely have the trailing underscore and fix diff-bench later.